app/models/distribution.rb

```diff
@@ -46,7 +46,6 @@ class Distribution < ApplicationRecord
 
   enum state: { scheduled: 5, complete: 10 }
   enum delivery_method: { pick_up: 0, delivery: 1, shipped: 2 }
-  scope :active, -> { joins(:line_items).joins(:items).where(items: { active: true }) }
```

Comment:
I can remove this change if preferred. But this looks like a piece of dead code.